charts/netdata/Chart.yaml

```diff
@@ -1,15 +1,28 @@
-apiVersion: v1
+apiVersion: v2
```

Comment:
@knatsakis have you considered that this may break people using helm < 3.0 ? I'm not sure it will, but thought I'd mention it.

Reply:
I have reverted `apiVersion` to v1.

Let upgrade to v2 after EOL of v1 this November: https://helm.sh/blog/helm-v2-deprecation-timeline/

Reply:
Opened #173 for this